docs/modules/ROOT/pages/how-to/organizations.adoc

```diff
@@ -0,0 +1,45 @@
+= Managing Organizations
+
+One of the basic features of the {product} Portal is to provide a centralized location for users to manage access to their projects across all {product} zones.
+
+== Concepts
+
+{product} allows namespaces and projects to belong to a particular {product} organization, as explained in the https://docs.appuio.cloud/user/how-to/manage-projects-and-namespaces.html[Manage Projects and Namespaces] documentation page. On the other hand, the https://control.vshn.net/[VSHN Control Panel] also provides customer management features. Finally, OpenShift also provides a notion of Groups. How are all of these concepts related?
+
+{product} Organizations aren't related to VSHN Control Panel customers nor to OpenShift groups. This makes them independent and cross-zones; this means that a {product} Organization spans across {product} zones automatically, allowing users to grant or deny access to projects from a central location.
+
+== Finding your Username
+
+In order to be able to assign users to organizations, users need to know their username. Users not assigned to an organization are reminded to join one as soon as they log in to the {product} Portal for the first time.
+
+image::how-to/organizations/welcome.png[]
+
```

Comment:
Alternatively, the username can be found in the user menu (currently bottom left, will be top right in the future)..

Reply:
Added separate page showing this